footer.js

```diff
@@ -0,0 +1,63 @@
+const repo = "todaysCuriosity";
+const owner = "mrhut10";
+const requestURL =
+  "https://api.github.com/repos/" + owner + "/" + repo + "/contributors";
+
+const contributors = [];
+
+const request = new XMLHttpRequest();
+request.open("GET", requestURL);
+request.responseType = "json";
+request.send();
+
+request.onload = function() {
+  request.response.forEach(contributor => {
+    contributors.push({
+      avatar_url: contributor.avatar_url,
+      login: contributor.login,
+      html_url: contributor.html_url,
+      contributions: contributor.contributions
+    });
+  });
+
+  const footerText = document.createElement("p");
```

Comment:
I have few points here about about semantic HTML, the layout of the contributes and also a point about readability.

 ### 1) semantic HTML
I don't like the fact that all of this has been put into a `<p>` tag. It's not good for accessibility/ screen readers. I would much prefer if they were instead in a `<ul>` with each element in a `<li>`. It makes more sense for what we're doing, listing out a number of people.

### 2) layout
I would like it if the contributors usernames and they're avatars were grouped together, as per the last point if the images and the user names were in the same `<ul>` that would be perfect.

### 3) readability

I'm not a big fan of the `document.createElement(. . .` followed by `.appendChild(. . . ` etc. It's hard to get an idea of what the output HTML is going to look like without really mulling over what you were doing here. I would prefer you created your HTML by using template literals (i.e. backticks) so it might look something like.
```javascript
contributorHtml = `
  <li>
    <a href="${contributor.html_url}">
      <img src="${contributor.avatar_url}">
      <span>${contributor.login}${contributor.contributions}</span>
    </a>
  </li>
'
```
To me this makes it very clear as to what you are injecting each loop.

Reply:
Recreated the html using template literals. It also reduce the number of lines of code overall. I agree, it's more readable.